exonum/src/api/public/blockchain_explorer.rs

```diff
@@ -16,25 +16,46 @@ use router::Router;
 use iron::prelude::*;
 
 use blockchain::{Blockchain, Block};
-use explorer::{BlockInfo, BlockchainExplorer};
+use explorer::{BlockInfo, BlockchainExplorer, TxInfo};
+use node::state::TxPool;
 use api::{Api, ApiError};
+use crypto::Hash;
 use helpers::Height;
 
 const MAX_BLOCKS_PER_REQUEST: u64 = 1000;
 
+#[derive(Serialize)]
+#[serde(tag = "type")]
+enum TransactionInfo {
+    Unknown,
+    InPool(PoolTransactionInfo),
```

Comment:
Would it work if the variant looked like `InPool { content: serde_json::Value }`? Or is this form chosen for the sake of future extensibility?